packages/fxa-auth-server/lib/routes/subscriptions/paypal.ts

```diff
@@ -350,9 +364,18 @@ export class PayPalHandler extends StripeWebhookHandler {
       }
     }
 
+    const sourceCountry =
+      customer.tax?.location?.country || customer.address?.country;
+
+    // if (!sourceCountry) {
+    //   const agreementDetails = await this.paypalHelper.agreementDetails({
+    //     billingAgreementId: customer.metadata.paypalAgreementId,
+    //   });
+    // sourceCountry = agreementDetails.countryCode
+    // }
```

Comment:
These lines will be removed before merging. They're the other way to resolve this issue, without simply removing the requirement that `countryCode` be set. This seems less desirable though, as I assume that the `countryCode` is expected to be set on the `customer` object